python/ray/dashboard/dashboard.py

```diff
@@ -60,7 +62,9 @@ def __init__(self,
         self.temp_dir = temp_dir
         self.node_stats = NodeStats(redis_address, redis_password)
 
-        self.app = aiohttp.web.Application(middlewares=[self.auth_middleware])
+        self.is_dev = os.environ.get("RAY_DASHBOARD_ENV") == "development"
```

Comment:
- comment about how `self.is_dev` is used.
- usage about `RAY_DASHBOARD_ENV` should go into readme.md

Reply:
Added a comment about `self.is_dev`, will update the README in the next batch of commits.